src/GraphControl/DirectX/RenderMain.cpp

```diff
@@ -157,19 +157,20 @@ namespace GraphControl::DX
             critical_section::scoped_lock lock(m_criticalSection);
 
             int formulaId = -1;
-            float nearestPointLocationX, nearestPointLocationY;
-            double nearestPointValueX, nearestPointValueY, rhoValueOut, thetaValueOut, tValueOut;
+            float nearestPointLocationX, nearestPointLocationY, nearestPointValueX, nearestPointValueY;                        
```

Comment:
Is there a version of the API where they are all doubles?

Reply:
there are, but they appear to be deprecated, the ones that have doubles as args don't have the precision parameter.